### PR TITLE
JanDeDobbeleer.OhMyPosh version 16.0.0

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106188)

### CODE REVIEW

**Comment by JanDeDobbeleer** [comment]
> a known installer technology like NullSoft, Inno, etc. the wingetcreate tool can detect them and the winget client will know what switches to pass

So, my question is as I add custom installer switches, do they then override all automatically detected switches? Because in that case I understand what's happening.

**Comment by JanDeDobbeleer** [comment]
@ImJoakim yup, but I'd expect it to deduct those from the InstallerType and scope just like it can for silent installs as well. I'll adjust!

File: manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-arm64.exe
+  InstallerSha256: 3774072E05D6C37AA27555BFDEF1CF790AC2402C8A5D3C153DA399599A938C05
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-arm64.exe
+  InstallerSha256: 3774072E05D6C37AA27555BFDEF1CF790AC2402C8A5D3C153DA399599A938C05
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:50]:
```suggestion
    Custom: /INSTALLER=winget

```

Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:50]:
```suggestion
    Custom: /INSTALLER=winget /CurrentUser
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-386.exe
+  InstallerSha256: BB50097CE7AE829C8D371B8155FEDDDC8DEA500B53F01D0D4DE993559F09A63E
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-386.exe
+  InstallerSha256: BB50097CE7AE829C8D371B8155FEDDDC8DEA500B53F01D0D4DE993559F09A63E
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:38]:
```suggestion
    Custom: /INSTALLER=winget /CurrentUser
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-amd64.exe
+  InstallerSha256: 4459CC4002D98F0B88F46356C197934039BE33E4065190E689C7F9CED354B8A7
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:20]:
```suggestion
    Custom: /INSTALLER=winget /AllUsers
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-386.exe
+  InstallerSha256: BB50097CE7AE829C8D371B8155FEDDDC8DEA500B53F01D0D4DE993559F09A63E
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:32]:
```suggestion
    Custom: /INSTALLER=winget /AllUsers
```

File: manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+- Architecture: arm64
+  Scope: machine
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v16.0.0/install-arm64.exe
+  InstallerSha256: 3774072E05D6C37AA27555BFDEF1CF790AC2402C8A5D3C153DA399599A938C05
+  InstallerSwitches:
+    Custom: /INSTALLER=winget
```
Comment (stephengillie) [inline manifests/j/JanDeDobbeleer/OhMyPosh/16.0.0/JanDeDobbeleer.OhMyPosh.installer.yaml:44]:
```suggestion
    Custom: /INSTALLER=winget /AllUsers
```

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**